README.md

```diff
@@ -1,2 +1,57 @@
+[![Build Status](https://travis-ci.org/dockstore/data-object-service-plugin.svg?branch=master)](https://travis-ci.org/dockstore/data-object-service-plugin)
+[![Coverage Status](https://coveralls.io/repos/github/dockstore/data-object-service-plugin/badge.svg?branch=master)](https://coveralls.io/github/dockstore/data-object-service-plugin?branch=master)
+
 # data-object-service-plugin
-Dockstore Data Object Service file provisioning plugin
+Dockstore Data Object Service file preprovisioning plugin
+
+## Usage
+
+The Data Object Service plugin is capable of calling other Dockstore plugins (i.e. syn, s3, icgc, etc.) to download, upload, and set metadata on uploaded objects.
```

Comment:
The plugin is currently download only. You shouldn't mention upload and setting metadata, and I think you should explicitly state that it is download only, mentioning that upload support may come in the future.